vispy/testing/_runners.py

```diff
@@ -111,6 +111,25 @@ def _unit(mode, extra_arg_string, coverage=False):
         os.rename(op.join(cwd, '.coverage'), out_name)
 
 
+def _docs():
+    """test docstring paramters
+    using vispy/utils/tests/test_docstring_parameters.py"""
+    print("warning: this imports vispy from the vispy installation. it is"
+          " recommended to setup vispy using 'python setup.py develop'"
+          " so that the latest sources are used automatically")
```

Comment:
Actually we have a heuristic to detect if someone is using a dev version or not, you should only throw a warning `if not _get_import_dir()[1]` (and use `warnings.warn` instead of print)